hi-ml-cpath/testhisto/testhisto/datamodules/test_slides_datamodule.py

```diff
@@ -38,7 +39,7 @@ def mock_panda_slides_root_dir(
         src_data_path=tmp_path_to_pathmnist_dataset,
         mock_type=MockHistoDataType.PATHMNIST,
         n_tiles=1,
-        n_slides=10,
+        n_slides=16,
```

Comment:
increased to be able to have more than one slide for testing/validation. The current split is .8 .1 .1